library/roc_consumer/src/request/request.e

```diff
@@ -0,0 +1,207 @@
+note
+	description: "Summary description for {REQUEST}."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
```

Comment:
note that I am still not sure to understand the need for this class.
Can't you use directly http_client directly?

it seems an extra layer on top of http_client .. but prevent for instance persistent connection reuse.